packages/enzyme/src/Utils.js

```diff
@@ -320,3 +320,49 @@ export function spyMethod(instance, methodName) {
     },
   };
 }
+
+export function validEmptyValues(renderedValue) {
+  return renderedValue === null || renderedValue === false;
+}
+
+export function renderedDive(nodes) {
+  let isEmptyRender = true;
+  const renderedKey = 'rendered';
+
+  function dive(renderedNodes) {
+    const isList = renderedNodes && renderedNodes.length > 0 && Array.isArray(renderedNodes);
+    if (isList) {
+      return renderedNodes.some((n) => {
+        if (n) {
+          if ((has(n, renderedKey))) {
+            dive(n.rendered);
+          } else if (!validEmptyValues(n.rendered)) {
```

Comment:
also here?

Reply:
if we remove renderedKey could we keep this?

Reply:
yeah def; we should either use renderedKey everywhere, or nowhere